client.go

```diff
@@ -11,21 +11,25 @@ type client struct {
 }
 
 const (
-	spaceLookUpMaxIDs         = 100
-	tweetLookUpMaxIDs         = 100
-	userLookUpMaxIDs          = 100
-	tweetSearchMaxQueryLength = 512
+	spaceLookUpMaxIDs           = 100
+	tweetLookUpMaxIDs           = 100
+	userLookUpMaxIDs            = 100
+	tweetSearchMaxQueryLength   = 512
+	filteredStreamRuleMaxLength = 512
 )
 
 type Client interface {
 	// CountsFullArchiveTweet(ctx context.Context, query string, opt ...*TweetCountsOption) (*TweetCountsResponse, error)
+	AddOrDeleteRules(ctx context.Context, body *AddOrDeleteJSONBody, opt ...*AddOrDeleteRulesOption) (*AddOrDeleteRulesResponse, error)
+	ConnectToStream(ctx context.Context, n int, opt ...*ConnectToStreamOption) (*ConnectToStreamResponse, error)
```

Comment:
`ConnectToStream` must be improved.
Because, The method's actual execution is stream.
So, I suggest to use channel.

Reply:
FYI
https://github.com/sivchari/gotwtr/pull/63/files